helpers/dom.js

```diff
@@ -227,3 +227,19 @@ export function isVisible(node) {
 	return true;
 
 }
+
+export function querySelectorComposed(node, selector) {
+
+	const elem = node.querySelector(selector);
+	if (elem) return elem;
+
+	const allElems = node.querySelectorAll('*');
```

Comment:
It's a shame there's no query selector for just custom elements, but I didn't find anything.